librz/cons/histogram.c

```diff
@@ -207,3 +210,133 @@ RZ_API RZ_OWN RzStrBuf *rz_histogram_vertical(RZ_NONNULL RzHistogramOptions *opt
 	}
 	return buf;
 }
+
+RZ_API RzHistogramOptions *rz_cons_histogram_options_new() {
+	RzHistogramOptions *histops = RZ_NEW0(RzHistogramOptions);
+	if (!histops) {
+		return NULL;
+	}
+	return histops;
+}
+RZ_API void rz_cons_histogram_options_free(RzHistogramOptions *histops) {
+	free(histops);
+}
+RZ_API RzIHistogram *rz_i_histogram_new(RzConsCanvas *can, RzHistogramOptions *opts) {
```

Comment:
Rename all the `rz_i_histogram_` into `rz_cons_i_histogram_new`

Reply:
In fact, a better name would be `rz_cons_histogram_interactive_new()`

Reply:
Or, better without `rz_cons_`, just like original `rz_histogram_horizontal()`, for example. Because the idea was to move it outside of the RzCons eventually since it doesn't really need anything except palette